codec-http2/src/test/java/io/netty/handler/codec/http2/Http2ConnectionHandlerTest.java

```diff
@@ -691,6 +691,14 @@ public ChannelFuture answer(InvocationOnMock invocation) throws Throwable {
         verify(pipeline).fireExceptionCaught(cause);
     }
 
+    @Test
+    public void canCloseStreamWithVoidPromise() throws Exception {
+        handler = newHandler();
+        handler.closeStream(stream, ctx.voidPromise());
+        verify(stream, times(1)).close();
+        verifyNoMoreInteractions(stream);
```

Comment:
Based on the other tests I wanted to do something like this:
```java
        verify(voidPromise, never()).addListener(any(GenericFutureListener.class));
        verify(ctx.channel(), times(1)).newPromise();
        verify(promise, times(1)).addListener(any(GenericFutureListener.class));
```
I'm not used to this testing suite, so please advise how can I get these checks to work... :pray: